addons/stock/static/src/legacy_web_report/report_backend.scss

```diff
@@ -1,9 +1,4 @@
 .o_report_iframe {
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
```

Comment:
I can't remember, but I think it did either nothing, or was no longer required because of the new DOM.
Doesn't really matter it's meant to disappear anyway.

Reply:
I'm afraid it won't disappear in v16, so it's still important. I would keep the moved code as is, for two reasons:
1) it simplifies the reading of the diff (you just move code)
2) you don't take any risk to introduce issues

Actually, the argument "it's meant to disappear anyway" is a good reason not to change anything.